src/components/input/input.html

```diff
@@ -35,7 +35,7 @@
 
       <label class="md-input-placeholder"
              [attr.for]="inputId"
-             [class.md-empty]="empty"
+             [class.md-empty]="empty && type !== 'date'"
```

Comment:
At thx, yeah that's a difference, since this is just an inline check. And the given variable / getter is only returning the value and not the boolean. `empty` is actually a property, which returns a boolean, which means that the check with the `date` is part of the is `empty` check.